middleware/src/middleware.go

```diff
@@ -136,6 +137,9 @@ func (middleware *Middleware) GetVerificationProgress() bool {
 // rpcLoop gets new data from the various rpc connections of the middleware and emits events if new data is available
 func (middleware *Middleware) rpcLoop() {
 	for {
+		if middleware.didServiceInfoChange() {
+			middleware.events <- []byte(rpcmessages.OpServiceInfoChanged)
+		}
 		if middleware.GetSampleInfo() {
```

Comment:
Lets finally retire the last of the sample/demo calls. Just need to make sure that they are properly removed from the app frontend as well.

Reply:
Yeah. Will do that in a separate PR as discussed with Stadicus a few days ago. Enables us to merge whenever the App stops using the old demo calls.